specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json

```diff
@@ -1417,6 +1417,51 @@
         }
       }
     },
+    "/providers/Microsoft.CostManagement/CheckConnectorEligibility": {
```

Comment:
[nit] please use camel case for resource segments in the URI (`checkConnectorEligibility` instead of `CheckConnectorEligibility`)

Reply:
done